src/translations/index.js

```diff
@@ -237,6 +240,7 @@ const TRANSLATIONS = {
         'passwordbox-repeat-password-long': 'Keine Übereinstimmung, bitte versuche es erneut',
         'passwordbox-repeat-password-short': 'Das Passwort ist zu kurz',
         'passwordbox-password-skip': 'Erstmal überspringen',
+        'passwordbox-reset-password': 'Mit Wiederherstellungswörtern zurücksetzen',
```

Comment:
This german translation is propably too long for the passwordbox to fit into one line...

Reply:
That is likely going to be an issue in many more places. If you don't have a good short suggestion I suggest doing it when the translation is actually being used.